cmd/plugin/builder/template/plugintemplates/plugin-tooling.mk.tmpl

```diff
@@ -22,6 +22,13 @@ PLUGIN_LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-plugin-runtime/plugin/build
 PLUGIN_LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-plugin-runtime/plugin/buildinfo.Version=$(PLUGIN_BUILD_VERSION)'
 PLUGIN_GO_FLAGS ?=
```

Comment:
After the latest commit, this variable is looking a little cryptic. A comment explaining its purpose, maybe accompanied with an example (like gcflag settings to _re-enable_ some level of function inlining), could be helpful.